notebooks/faq.clj

```diff
@@ -0,0 +1,62 @@
+
+More concretely, the Wolfram language, and associated ecosystem, is a powerful way of getting stuff done. As a high-level functional programming language, with more than 6000 functions from across mathematics and data processing, it is sometimes the only practical way of of getting things done. Particularly in explicitly scientific fields. What makes it potentially unique is its place in the Venn diagram of symbolic manipulation, built-in factual and computational knowledge and niche, fully implemented algorithms.
+
+**Why Wolframite?**
+
+'Why Wolframite?' is really just the combination of 'Why Wolfram?' (above) and '[Why Clojure?](https://clojure.org/about/rationale)'.
+
+In a nutshell, the Wolfram programming language is 'best-in-class' for certain specialist tasks, like manipulating equations, but, to paraphrase (Randall Munroe)[https://xkcd.com/1270/],
+> Wolfram combines the flexibility and power of abstract mathematics with the intuitive clarity of abstract mathematics.
+
+Clojure, on the other-hand, is a first-class, general-purpose, programming language whose core namespace and dynamical, functional, paradigm is well suited to data exploration and manipulation. So why don't we call Wolfram from Clojure? It seems like the atypical 'best of both worlds'.
+
+To put it another way, Wolfram is *a powerful tool in need of a toolbox*. And so what language is best placed to be that 'box'? Well, Wolfram is, underneath, built on LisP and so will naturally gel more with a similar design philosophy (and way of thinking) than, say, with an object-oriented or scripting language. Furthermore, somewhat ironically, it has a very good 'link' to the Java ecosystem. In [fact](https://reference.wolfram.com/language/JLink/tutorial/Introduction.html#25629),
+>  For all types of WSTP programs, J/Link provides a higher-level layer of functionality than the traditional C WSTP programming interface. This makes Java the easiest and most convenient language for writing programs to interact with the Wolfram Language.
+
+Even at a fundamental level therefore, our toolbox language should be a LisP with strong Java interop: mmm...!
+
+At the *usability* layer, Clojure is a well-designed, ergonomic language that leaves its users [happy](https://www.computerworld.com/article/1379902/clojure-developers-are-the-happiest-developers.html)! When it comes to working with mathematical expressions specifically, there are a few key features. First of all, for many people, Clojure was the first language to introduce ergonomic chaining of function expressions, *e.g.*
+```clojure
+(TeX->
+   (w/* 'x w/I)
+   (w/+ 'y)
+   ||2)
+```
```

Comment:
wouldn't this be better as evaluated code? I have no idea what it does 😅 Also, what does the `||2` dp?!

Reply:
updated